easy-coding-standard.neon

```diff
@@ -31,6 +31,7 @@ checkers:
             - Symplify\BetterReflectionDocBlock\*
 
 parameters:
+    cache_directory: .ecs_cache
```

Comment:
Please keep defaults as it is now, since  it's most user friendly approach that does not polute repository with temp files.
Like Kernel does not

Reply:
I don't think that `/tmp/_changed_files_detector` would be safe to use as default here now. Some OS'es might not even have that directory.

Thanks for the comments, I'll push the requested changes over the weekend.

Reply:
I see. Do you think local `/temp` in the project is way to go?

Imagine you'd install this package as dependency and it would create `/temp` or `/tmp` or anyother named directory in your root.

Reply:
The default behaviour stays the same. If user does not provide cache_directory argument in his neon config, the sys_get_temp_dir() approach will be used in CacheFactory. The .ecs_cache is for this project only (Symplify/Symplify) as an example.

Reply:
I see. Example in README is good enough